go/env/main.go

```diff
@@ -75,6 +75,13 @@ func setupEnv(ctx context.Context, repo *github.Repository) error {
 		return err
 	}
 
+	_, _, err = c.Repositories.CreateDeploymentBranchPolicy(ctx, repo.GetOwner().GetLogin(), repo.GetName(), *envName, &github.DeploymentBranchPolicyRequest{
```

Comment:
I'm starting to feel like we should use terraform for these repos...

https://registry.terraform.io/providers/integrations/github/latest/docs/resources/repository_deployment_branch_policy

Reply:
well.. now we have everything working, though I see your point. We can convert it later?